generator/generator_config.textproto

```diff
@@ -3238,7 +3238,13 @@ service {
   service_proto_path: "google/storage/control/v2/storage_control.proto"
   product_path: "google/cloud/storagecontrol/v2"
   initial_copyright_year: "2024"
-  retryable_status_codes: ["kUnavailable"]
+  retryable_status_codes: [
+    "kResourceExhausted",
+    "kUnavailable",
+    "kDeadlineExceeded",
+    "kInternal",
+    "kUnknown"
+  ]
```

Comment:
`idempotency_overrides` doesn't include `StorageControl.RenameFolder` does that need to be manually changed as well?

Reply:
Also clarifying, Is this the default retriable codes for idempotent operations only?

Reply:
> `idempotency_overrides` doesn't include `StorageControl.RenameFolder` does that need to be manually changed as well?

No. In that case the autopopulated `request_id` will take effect.  The problem with the other RPCs is that they are inherently idempotent (they are read-only operations), but they don't have the HTTP annotations that C++ uses to determine if the operation should be idempotent.  And some of these functions don't have a `request_id` configured [^1].

[^1]: the field exists, but the YAML file does not include it.

Reply:
thanks for clarifying!

Reply:
> Also clarifying, Is this the default set of retriable codes for idempotent operations only?

yes.